src/v2/Apps/ViewingRoom/Routes/Statement/Components/ViewingRoomBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { Box, Serif } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ViewingRoomBody_viewingRoom } from "v2/__generated__/ViewingRoomBody_viewingRoom.graphql"
+
+interface ViewingRoomBodyProps {
+  viewingRoom: ViewingRoomBody_viewingRoom
+}
+
+const ViewingRoomBody: React.FC<ViewingRoomBodyProps> = ({
+  viewingRoom: { body },
+}) => {
+  if (!body) {
+    return null
+  }
+
+  return (
+    <Box>
+      <Serif size={["4", "5"]} style={{ whiteSpace: "pre-line" }}>
```

Comment:
Not sure what is our best practices for supporting new lines in Html format. For now just putting this `whiteSpace` inline. Will probably need to refactor that later.